File: JAVA SCRIPT/06.js
//6. Write a function that checks if a username and password match predefined values. If the username is "admin" and the password is "1234", print "Login successful"; otherwise, print "Login failed."
function login(username,password){
    if(username=="admin" && password==1234){
        console.log("login successful")
    }
    else{
        console.log("login fail");
    }
}
login("admin",1234);

File: C LANGUAGE/06.c
//6. Write a function that checks if a username and password match predefined values.
	#include<stdio.h>
	int main(){
		char username[]="admin";
		char name[]="admin";
		int count=0;
		int password=1234;
		int length=sizeof(username)/sizeof(username[0]);
		for(int i=0;i<length;i++){
			if(username[i]==name[i] && password==1234){
				 count++;
			}
		}
		if(count==length){
			printf("login successful");
		}
		else{
			printf("login fail");
		}
	}
